tests/baselines/reference/api/typescript.d.ts

```diff
@@ -3525,6 +3525,7 @@ declare namespace ts {
     function isCallExpression(node: Node): node is CallExpression;
     function isCallChain(node: Node): node is CallChain;
     function isOptionalChain(node: Node): node is PropertyAccessChain | ElementAccessChain | CallChain;
+    function isQuestionQuestionExpression(node: Node): boolean;
```

Comment:
Why not

`isNullishCoalesceExpression(node: Node): node is NullCoalesceExpression`